outline

```diff
@@ -111,7 +111,7 @@ Fuse : articles/index.md
 		Path : api-docs/api/fuse/controls/path.json
 		Image : api-docs/api/fuse/controls/image.json
 		Video : api-docs/api/fuse/controls/video.json
-		// Where is this located? Sound : api-docs/api/fuse/controls/sound.json
+		Sound : api-docs/api/fuse/playsound.json
```

Comment:
Nice!

Reply:
So it worked! 😄

Reply:
Oh, I'm not sure I actually tested it! I'll check that next time I find some extra time for this :)

Reply:
Heehee, cool